src/plays/popup-component/components/popupCard.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+function popupCard({ children, popTitle, btnStyle }) {
+  const [toggle, setToggle] = useState(false);
+  const toggleModal = () => {
+    setToggle(!toggle);
+  };
+
+  return (
+    <>
+      <button className="modal__btn" style={btnStyle} onClick={toggleModal}>
+        {popTitle}
+      </button>
+      {toggle && (
```

Comment:
Instead of using the logical **AND** (`&&`) operator for conditional rendering, use a ternary operator for improved readability.
 
 This change makes it clearer that the toggle state controls the visibility of the modal content.